### ISSUE TITLE
Label Redesign

### ISSUE THREAD

**Issue by nsgnn** [ChristopherGerbi/capstone#23]
Labels should be external to the text input fields. They should also allow show the units of the field. Alt text of the text input fields mainly be example input format.
